### ISSUE TITLE
Configure a full hostname for Speech service containers

### ISSUE THREAD

**Issue by palancana** [microsoft/BotFramework-WebChat#3291]
## Feature Request

**Is your feature request related to a problem? Please describe.**

I'm unable to configure a hostname for a Speech service containers.

**Describe the suggestion or request in detail**

I have a Docker container running TTS service. The exposed URL for this container is something like http://127.0.0.1:5001. Following the documentation, [I found that I can modify `speechSynthesisHostname`](https://github.com/microsoft/BotFramework-WebChat/blob/master/docs/SPEECH.md#using-sovereign-cloud). As my docker container is running locally, I need to use **http://**, but [in the code, **https://** is hardcoded](https://github.com/microsoft/BotFramework-WebChat/blob/74250046370eb152c225c5be02e5f68fba17973d/packages/testharness/src/speech/speechRecognition/fetchSpeechData.js#L46), so there is no way to build the URL I need.

**Describe alternatives you have considered**

To keep current behaviour as a non breaking change, we could default to https and have another property or flag to specify either a full URL or force the usage of **http**.

[Enhancement]

**Comment by cleemullins** [comment]
Hardcoding HTTPS seems reasonable for us, although I see how that would cause you a problem. 

Can you create a self-signed cert, or get a localhost cert via LetsEncrypt or similar? Seems the quickest way to work around the issue.

**Comment by palancana** [comment]
Yes! Both approaches are valid. I already found a solution around with the self-signed certificate. If you don't think this enhancement would help other situations, we may close the ticket.

**Comment by compulim** [comment]
@palancana Can you point us to documentation on configuring speech containers?

**Comment by dmvtech** [comment]
Just adding a few more _potential_ workarounds (not related to enhancement); 

1. Cloning, modifying and building your own webchat
2. Using something like Ngrok or Service Bus (not likely a worthwhile production solution).

**Comment by palancana** [comment]
> @palancana Can you point us to documentation on configuring speech containers?

Here is the [link](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/speech-container-howto?tabs=tts%2Ccsharp%2Csimple-format).

> 1. Cloning, modifying and building your own webchat
> 2. Using something like Ngrok or Service Bus (not likely a worthwhile production solution).

That's another approach, yes. Thanks for contributing!

**Comment by munozemilio** [comment]
JJ will take a look into this one

**Comment by dmvtech** [comment]
@compulim Is this something you want to take as an enhancement/DCR? Seems like there are more than a few workarounds. Can you please close if not (I don't currently have perms to do so).

**Comment by carlosscastro** [comment]
@compulim could you please evaluate Dana's suggestion on whether we should close this issue and / or take as an enhancement?
